packages/visualizations/addon/validators/request-metric-exist.js

```diff
@@ -5,13 +5,14 @@
 
 import Ember from 'ember';
 import BaseValidator from 'ember-cp-validations/validators/base';
+import isEqual from 'lodash/isEqual';
 import { getRequestMetrics } from 'navi-visualizations/utils/chart-data';
 
 const { A:arr, getWithDefault } = Ember;
 
 export default BaseValidator.extend({
   validate(value, options/*, model, attribute*/) {
     let requestMetrics = getRequestMetrics(getWithDefault(options, 'request', {}));
-    return arr(requestMetrics).includes(value);
+    return !!arr(requestMetrics).find(metric => isEqual(metric, value));
```

Comment:
`isAny`?

Reply:
`isAny` is for checking for a certain property in a list of objects right? We want to compare the entire object, not just one property.

Reply:
Why are we checking the entire object? Would checking the canonical name not be enough?